src/test/java/com/elmsoftware/env/EnvironmentSettingsModuleTest.java

```diff
@@ -118,17 +97,23 @@ public void should_use_settings_provider_if_provided() throws Exception {
 		assertNotNull(providerField);
 		providerField.setAccessible(true);
 		assertNotNull(providerField.get(module));
-
 	}
 
-	private Key<String> findKey(final List<Key<String>> allValues, final String name) {
-		for (final Key<String> key : allValues) {
-			final Named annotation = (Named) key.getAnnotation();
-			if (annotation.value().equals(name)) {
-				return key;
-			}
-		}
-		return null;
-	}
+	@Test
+	public void should_use_environment_settings_if_set() throws Exception {
+		// setup test
+		final String dockerEnvironmentVariable = "I'm configurable for docker!";
+		final EnvironmentSettingsModule module = new EnvironmentSettingsModule(new NoOpSettingProvider(), util);
 
+		System.setProperty("environment.json", "guice-test.json");
+		when(binder.bind(any(Key.class))).thenReturn(builder);
+		when(binder.skipSources(Names.class)).thenReturn(binder);
+
+		// run test
+		withEnvironmentVariable("SOME_KEY", dockerEnvironmentVariable)
```

Comment:
im usually not a fan of bringing in a new dependency for one test, but in this case it seemed easiest.